File: TP1/c/operateurs.c
/* Fichier: operateurs.c
* Utilisation des opérateurs arithmétiques et logiques en C
* auteur: Pierre MOLY
* Exercice 1.5
*/

#include <stdio.h>
#include <stdbool.h>
int main(){

    int a = 16;
    int b = 3;

    int calcul = a + b;
    printf("a + b = %d\n", calcul);

    calcul = a - b;
    printf("a - b = %d\n", calcul);

    calcul = a * b;
    printf("a * b = %d\n", calcul);

    float division = (float)a / b;
    printf("a / b = %f\n", division);

    calcul = a % b;
    printf("a %% b = %d\n", calcul);

    bool boolean = a == b;
    printf("a == b = %d\n", boolean);

    boolean = a > b;
    printf("a > b = %d\n", boolean);

    return 0;
}

File: TP1/python/operateurs.py
# Fichier: operateurs.py
# Utilisation des opérateurs arithmétiques et logiques en Python
# auteur: Pierre MOLY
# Exercice 1.5

def main():
    # affectation des variables
    a = 16
    b = 3

    # calculs et affichage des résultats
    calcul = a + b
    print("a + b = ", calcul)

    calcul = a - b
    print("a - b = ", calcul)

    calcul = a * b
    print("a * b = ", calcul)

    calcul = a / b
    print("a / b = ", calcul)

    calcul = a % b
    print("a % b = ", calcul)

    boolean = a == b
    print("a == b = ", boolean)

    boolean = a > b
    print("a > b = ", boolean)

if __name__ == "__main__":
    main()